apps/rush-lib/src/cli/test/__snapshots__/CommandLineHelp.test.ts.snap

```diff
@@ -231,6 +233,7 @@ Optional arguments:
   --ignore-hooks        Skips execution of the \\"eventHooks\\" scripts defined 
                         in rush.json. Make sure you know what you are 
                         skipping.
+  --disable-cache       Disables the build cache for this command invocation.
```

Comment:
If the cloud build cache is still experimental, then add `(EXPERIMENTAL) ` here.